docl/commands.py

```diff
@@ -53,14 +53,19 @@
 
 
 @command
+@argh.arg('-u', '--manager-image-url')
+@argh.arg('-m', '--manager-image-docker-tag')
```

Comment:
Is that in use?

Reply:
Oh yes, that's the way to set the docl image it'll be using (it always used a default value in the config, now I added a param to pass from the CLI)